gson/src/main/java/com/google/gson/internal/bind/DateTimeTypeAdapters.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2011-2016 Google Inc.
```

Comment:
I think you should pick either 2011 or 2016? As is it looks like the copyright expires at the end of 2016!